elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/ElasticsearchQueries.java

```diff
@@ -28,7 +27,7 @@ private ElasticsearchQueries() {
 	 * documentation</a> for the complete query syntax.
 	 */
 	public static QueryDescriptor fromJson(String jsonQuery) {
-		return new ElasticsearchJsonQueryDescriptor( new JsonParser().parse( jsonQuery ).getAsJsonObject() );
+		return new ElasticsearchJsonQueryDescriptor( GsonHolder.PARSER.parse( jsonQuery ).getAsJsonObject() );
```

Comment:
yes I noticed that, but I prefer your change (i.e. keep the static reference), it's just reassuring that in case it turns out the component will be leaking in future, then we can revert to instantiation.

BTW it's not the PARSER the one which uses tlocals, but the encoder (the "GsonHolder.GSON" )